plugin/src/sidebar.ts

```diff
@@ -353,6 +353,16 @@ reearth.on("message", ({ action, payload }: PostMessageProps) => {
     }
   }
 
+  // ************************************************
+  // Style code
+  else if (action === "styleCodeUpdate") {
+    const { dataID, styleObject } = payload;
+    const layerId = addedDatasets.find(ad => ad[0] === dataID)?.[2];
```

Comment:
layerID

Reply:
Ahhh.. updated. I guess i also use `layerId` in other components...